lua/autorun/server/sv_limit_explosives.lua

```diff
@@ -1,7 +1,12 @@
 local limits = {
-    m9k_proxy = 4
+    m9k_proxy = 4,
+    npc_satchel = 15
 }
 
+for _, k in ipairs( player.GetHumans() ) do
+    k.placedExplosives = nil
+end
```

Comment:
I don't think there will be any players when this file is loaded?

Reply:
oh mb that was for testing fml